book.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# create class Book
+class Book
+  attr_accessor :name, :author
```

Comment:
- [ ] `rental` (in your case _rents_) should be marked with `attr_accessor`, as we intend to both _access (get)_ as well as _write (set)_ it. Kindly ensure to improve upon this aspect.